syncopy/statistics/summary_stats.py

```diff
@@ -206,6 +208,9 @@ def itc(spec_data, **kwargs):
         act = "real valued spectral data"
         raise SPYValueError(lgl, 'spec_data', act)
 
+    logger = logging.getLogger("syncopy_" + platform.node())
+    logger.debug(f"Computing intertrial coherence on SpectralData instancewith shape {spec_data.data.shape}.")
```

Comment:
missing space between `'instance'` and `'with'`

Reply:
fixed